app/events.py

```diff
@@ -34,3 +36,10 @@ def delete(host):
         )
         .data
     )
+
+
+def emit_patch_event(host, metadata=None):
+    key = str(host.id)
+    metadata = {"request_id": threadctx.request_id}
+    event = build_event("updated", host, None, metadata)
```

Comment:
Or as suggested later on, we can ditch the metadata argument entirely.
```suggestion}
    event = build_event("updated", host, request_id=threadctx.request_id)
```